### ISSUE TITLE
edit single box

### ISSUE THREAD

**Issue by vungo99** [Tkko/Flutter_Pinput#150]
is available when pinput allows user to edit a single box?

**Comment by Tkko** [comment]
Hi @vungo99, it's not available yet.

**Comment by vungo99** [comment]
tks for your reply. one more question about the edge case, when 2 or more SMS OTPs are sent at the same time, can pinput detect correctly SMS OTP from the right app's vendor?
thank u so muchh

**Comment by Tkko** [comment]
@vungo99 If you enable smsRetrieverApi it will only read SMS that will have the application signature in the content. Check out the autofill section in the readme for more details.

**Comment by vungo99** [comment]
Can I use smsCodeMatcher to detect OTP when message has many number like : "your account 123456 will receive OTP 654321 and has valid in 300s " => detect otp : 654321

**Comment by Tkko** [comment]
@vungo99 Yes, you can set a regex that will match the OTP instead of account id. Do both OTP and account Id have the same length?

**Comment by vungo99** [comment]
Yes, they have the same length like :
" your id 022C123456 will receive OTP 666666 in 300s". 
I write regex OTP\\s*(\\d{6}) but not work. How can I edit it?

**Comment by Tkko** [comment]
@vungo99 Try this regex ` r'\b(\d{6})\b'`

**Comment by vungo99** [comment]
smsCodeMatcher: '\b(\d{6})\b', right ?

**Comment by Tkko** [comment]
```
smsCodeMatcher: r'\b(\d{6})\b'
```

**Comment by vungo99** [comment]
oh tk u so much !! It work like a charm :D

**Comment by vungo99** [comment]
hi @Tkko , I have the issue :
msg 1 : "TBC: your id 022C123456 will receive OTP 666666 in 300s "
msg 2 : "your id 022C123456 will receive OTP 666666 in 300s" 
how can use regex to accept otp from msg1, not msg2

**Comment by Tkko** [comment]
Hey @vungo99, I usually ask regex questions to ChatGPT 🙈

**Comment by vungo99** [comment]
I use it too, but can't apply 🙈🙈🙈
